src/plotman/analyzer.py

```diff
@@ -9,7 +9,8 @@
 
 
 class LogAnalyzer:
-    def analyze(self, logfilenames, bytmp, bybitfield):
+    @staticmethod
+    def analyze(logfilenames, bytmp, bybitfield):
```

Comment:
I personally just move stuff like this out as a freestanding function since it has no hard ties to the class.  Do you have any strong feelings for not doing that?

Reply:
Seems like a simple method would work just fine - I wasn't sure if there was any backstory to this standalone class (plans to refactor it to something more OOP, etc.) so I didn't want to touch the external interface.

Reply:
No particular reason for this to be a class method right now, no, it would be fine to move it out.

Reply:
Oh hey, it's even the only thing at all in the class.  `:]`  Yeah, change requested here...